client/src/components/formHelpers/FormFields.js

```diff
@@ -22,39 +25,46 @@ const propTypes = {
 };
 
 function FormFields(props) {
-  const { fields, options = {} } = props;
+  const { formFields, options = {} } = props;
   const {
     ignored = [],
     placeholders = {},
     required = [],
     types = {}
   } = options;
+
   return (
     <div>
-      {fields
-        .filter(field => !ignored.includes(field))
-        .map(name => (
-          <Field key={`${name}-field`} name={name}>
+      {formFields
+        .filter(formField => !ignored.includes(formField.name))
+        .map(formField => (
```

Comment:
```suggestion
        .map(({name, label}) => (
```
This should save a few characters.  It'll likely cause prettier to re-format, so I haven't added comments for all the rest.